src/FallbackExportAdapter.php

```diff
@@ -52,11 +53,19 @@ public function export(ExportConfig $exportConfig, string $csvFilePath): ExportR
                 $this->logger->info(sprintf('Exporting by "%s" adapter.', $adapter->getName()));
                 return $adapter->export($exportConfig, $csvFilePath);
             } catch (Throwable $e) {
-                $this->logger->warning(sprintf(
-                    'Export by "%s" adapter failed: %s',
-                    $adapter->getName(),
-                    $e->getMessage()
-                ));
+                if ($e instanceof AdapterSkippedException) {
```

Comment:
Pridal som novy typ exception `AdapterSkippedException` ... tym sa to bude spravne logovat, ze adapter nespadol, ale je vypnuty.